staging/src/k8s.io/apiserver/pkg/server/options/tracing.go

```diff
@@ -106,6 +106,7 @@ func (o *TracingOptions) ApplyTo(es *egressselector.EgressSelector, c *server.Co
 	}
 	tp := traces.NewProvider(context.Background(), sampler, resourceOpts, opts...)
 	c.TracerProvider = &tp
+	c.LoopbackClientConfig.Wrap(traces.WrapperFor(c.TracerProvider))
```

Comment:
is LoopbackClientConfig always set at this point? do we need to check for nil?

does this ever come from the ClientConfig you're already wrapping in staging/src/k8s.io/apiserver/pkg/server/options/coreapi.go?

Reply:
checked for nil.

LoopbackClientConfig is created separately from either deprecated_insecure_serving.go, or serving_with_loopback.go.  In both cases is is created without referencing the existing client config.